region/crateria/central.json

```diff
@@ -1730,9 +1701,38 @@
                   "obstacles": [
                     {
                       "id": "A",
-                      "requires": []
+                      "requires": [ "h_canDestroyBombWalls" ]
+                    }
+                  ]
+                },
+                {
+                  "name": "Come in Charged",
+                  "notable": false,
+                  "requires": [],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": [
+                        {"canComeInCharged": {
```

Comment:
There should be a `canComeInCharged` with `framesRemaining: 180` and  `shinesparkFrames: 0` as well, on both directions.